EIPS/eip-6492.md

```diff
@@ -135,6 +140,11 @@ contract UniversalSigValidator {
       try IERC1271Wallet(_signer).isValidSignature(_hash, sigToValidate) returns (bytes4 magicValue) {
         bool isValid = magicValue == ERC1271_SUCCESS;
 
+        // retry, but this time assume the prefix is a prepare call
+        if (!isValid && !tryPrepare && contractCodeLen > 0) {
```

Comment:
shouldn't this be `!isValid && tryPrepare && contractCodeLen > 0`

Reply:
No, because if we didn't `tryPrepare` then we should, but if we did then there isn't much more we could do and the verification is in fact failing